From 55154e8e9a79473f4ed5eb690ad602072d7908cd Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 19 May 2022 10:55:26 +0300
Subject: [PATCH] [SW-0] common: remove todo comment

wfe command is verified on pldm and ARC goes to sleep as expected,
so remove comment.

Change-Id: I660098caecd6ddd581703c54ee9089719549e07c
---

diff --git a/include/arcboot.h b/include/arcboot.h
index 0d18030..264db56 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -19,7 +19,6 @@
 #include "hl_sec_if.h"
 #include "spi.h"
 
-/* TODO: check whether below command puts ARC to sleep */
 #define wfe()	__asm__ __volatile__("sleep 0x11\n" : : )
 
 #ifndef __packed
